app/assets/javascripts/bind_event_to_preview_page.js

```diff
@@ -26,7 +46,13 @@ $(document).ready(function () {
     var title = $('#event_title').val() === '' ? 'Event title goes here' : $('#event_title').val();
```

Comment:
NITPICK: avoid evaluating a DOM obj repeatedly, `$('#event_title')` can be assigned to a variable and then used concurrently after.